api_app/db/repositories/operations.py

```diff
@@ -163,6 +163,11 @@ def get_operation_by_id(self, operation_id: str) -> Operation:
             raise EntityDoesNotExist
         return parse_obj_as(Operation, operation[0])
 
+    def get_my_operations(self, user_id: str) -> List[Operation]:
+        query = self.operations_query() + f' c.user.id = "{user_id}" AND c.status IN ("{Status.AwaitingAction}", "{Status.InvokingAction}", "{Status.AwaitingDeployment}", "{Status.Deploying}", "{Status.AwaitingDeletion}", "{Status.Deleting}", "{Status.AwaitingUpdate}", "{Status.Updating}", "{Status.PipelineRunning}") ORDER BY c.createdWhen ASC'
```

Comment:
yea i don't like it either - in the UI I use a collection of "in_progress_statuses" - I could bring that here too, then just join the array in the query, rather than listing every in-progress state by hand?